src/main/java/seedu/multitasky/model/StorageUserPrefs.java

```diff
@@ -0,0 +1,43 @@
+package seedu.multitasky.model;
+
+//@@author A0132788U
+/**
+ * Represents the default values in UserPrefs object that storage has access to.
+ */
+public interface StorageUserPrefs {
+
+    public String getEntryBookFilePath();
+
+    public void setEntryBookFilePath(String entryBookFilePath);
+
+    /**
+     * The getter and setter for the snapshot file path
+     */
+    public static String getEntryBookSnapshotPath() {
+        return UserPrefs.getEntryBookSnapshotPath();
```

Comment:
not recommended to use UserPrefs static methods here. this forces StorageUserPrefs interface to be unnecessarily dependent on UserPrefs. i think just doing a public static methodname(); is enough for all the methods in the interface, let the classes that implement the interface override how they want the method to be implemented.